### ISSUE TITLE
Do not open first message by default

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#6526]
### Is your feature request related to a problem? Please describe.

In my email workflow I leave messages unread if I have to take action later on. Right now the Mail app automatically opens the first message of my priority inbox, leaving that message as read even if I don't read it.

### Describe the solution you'd like

Do not open the message until I navigate to it.

### Describe alternatives you've considered

N/a

### Additional context

There might be another ticket about this same request but I'm unable to find it right now.

cc @jancborchardt @nimishavijay

**Comment by nimishavijay** [comment]
Agreed, in the Mail survey as well there were a couple of responses that found this annoying. 
The feature "Don't automatically mark mails as read" was requested a bunch as well, but this seems like a much better fix.

**Comment by meichthys** [comment]
The first message is also opened after moving a message out of the current folder. 
Ideally the next (or previous) message in the current folder should be opened.

**Comment by jancborchardt** [comment]
@meichthys could you open a separate issue for that? Thank you! :)

**Comment by meichthys** [comment]
@jancborchardt I'll hold off on the new issue for now. I can't seem to reproduce it right now in v1.14.0-beta1

**Comment by slu-010101** [comment]
Still an issue with mail app 2.2.4.

**Comment by meichthys** [comment]
I can also confirm that moving a message to a different folder opens the first message

**Comment by meichthys** [comment]
@jancborchardt The related issue is filed here: https://github.com/nextcloud/mail/issues/8200
I can still reproduce it, but only when dragging a message (not when using the 'move thread' menu item)
